Assets Precompile in Rails Guide is NOT showing the steps for Sprockets 4 (#46304)
rails/rails: guides/source/asset_pipeline.md

`.scss` files are **not** automatically included as they compile to JS/CSS.

If you have other manifests or individual stylesheets and JavaScript files to
include, you can add them yourself:

> - **Sprockets 3:** Add files to `precompile` array in `config/initializers/assets.rb`:
>   ```ruby
>   Rails.application.config.assets.precompile += %w( admin.js admin.css )
>   ```
> - **Sprockets 4:** Add files or directories inside `./app/assets/config/manifest.js`
>   ```js
>   //= link_tree ../images
>   //= link_directory ../javascripts .js
>   //= link_directory ../stylesheets .css
>   ... (other files/directories)
>   ```

Reviewer: ```suggestion
- **Sprockets 3:** Add files to `precompile` array in `config/initializers/assets.rb`:

  ```ruby
  Rails.application.config.assets.precompile += %w( admin.js admin.css )
  ```

- **Sprockets 4:** Add files or directories inside `./app/assets/config/manifest.js`

  ```js
  //= link_tree ../images
  //= link_directory ../javascripts .js
  //= link_directory ../stylesheets .css
  ... (other files/directories)
  ```
```

Author: Just pushed the changes, also noticed that .gitignore does not have .DS_Store included, is there a reason for it?

`.scss` files are **not** automatically included as they compile to JS/CSS.

If you have other manifests or individual stylesheets and JavaScript files to
include, you can add them yourself:

> - **Sprockets 3:** Add files to `precompile` array in `config/initializers/assets.rb`:

Reviewer 2: ```suggestion
- **Sprockets 3:** Add files to the `precompile` array in `config/initializers/assets.rb`:
```

  ```ruby
  Rails.application.config.assets.precompile += %w( admin.js admin.css )
  ```

- **Sprockets 4:** Add files or directories inside `./app/assets/config/manifest.js`

>   ```js
>   //= link_tree ../images
>   //= link_directory ../javascripts .js
>   //= link_directory ../stylesheets .css
>   ... (other files/directories)
>   ```

Reviewer 2: I think it is clearer if we use the same admin assets in the example:
```suggestion
  ```js
  //= link_tree ../images
  //= link admin.js
  //= link admin.css
  //= link_directory ../javascripts .js
  //= link_directory ../stylesheets .css
  ```
```

NOTE. Always specify an expected compiled filename that ends with `.js` or `.css`,
even if you want to add Sass or CoffeeScript files to the precompile array or manifest file.

> NOTE. For more information in the differences between Sprockets 3 and 4 please go [here](https://github.com/rails/sprockets/blob/main/UPGRADING.md#manifestjs)

Reviewer 2: Instead of using `here` we can already describe what it links to:
```suggestion
NOTE. More information on the differences between Sprockets 3 and 4 please can be found in the [Sprockets Upgrading Guide](https://github.com/rails/sprockets/blob/main/UPGRADING.md#manifestjs)
```
